tools/AutoTuner/src/autotuner/distributed.py

```diff
@@ -336,6 +335,13 @@ def parse_arguments():
         default="",
         help="Additional arguments given to ./build_openroad.sh.",
     )
+    parser.add_argument(
+        "--cloud_dir",
+        type=str,
+        metavar="<str>",
+        default=None,
+        help="Cloud storage directory for logs, defaults to None. Currently supports only GCP.",
```

Comment:
What do we need to change to support other cold environments? My understanding was that as long as the Ray `yaml` files were correctly configured to a cloud environment supported by Ray, our code would work.

Reply:
I assume you mean other cloud environments. Your understanding is correct, we should just need to have appropriate configs (region, machine type, storage bucket, iam role to provision machine)

Reply:
(It is `cloud` indeed, I fixed the typo)

So let's change the wording here to:
```suggestion
        help="Cloud storage directory for logs, defaults to None. See our documentation on cloud support for more information.",
```
Add such a section and explain that we _should_ be able to support any cloud environment that Ray supports (link to their docs) but that, so far, we have only tested with GCP.